README.md

```diff
@@ -1,16 +1,34 @@
-# Microservices Demo
+# Modernized Microservices Demo
```

Comment:
@jvc5546 please review the README.md and modify accordingly. The README should describe the application and what it does, not the updates the agent did.